docs/paymentMethods/cardNotPresent.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Card Not Present (CNP) Transactions
+sidebar_label: Card Not Present (CNP)
+description: Learn about Card Not Present payment methods
+keywords:
+  - paymet methods
+  - card not present
+  - cnp
+---
+
+# Card Not Present (CNP) Transactions
+
+## Description
```

Comment:
I think you can remove this headline. The description below can just be under the main headline.